sdk/python/packages/flet-core/src/flet_core/control.py

```diff
@@ -515,6 +516,8 @@ def _build_add_commands(self, indent=0, index=None, added_controls=None):
                 indent=indent + 2, index=index, added_controls=added_controls
             )
             commands.extend(childCmd)
+            # set the parent
+            control.parent = self
```

Comment:
That function where it's set is recursive, so `control.parent` will be set to a top most control which `update()` is called for.

Reply:
What is your suggestion?